resources/scigraph/cypher-resources.yaml

```diff
@@ -1,12 +1,73 @@
 cypherResources:
-  - path: /dynamic/prod/sparc/organParts/{id}
+  - path: /dynamic/prod/sparc/organParts
     operations: &organParts-ops
+      - summary: Get the list of all FMA organ parts for all FMA organs relevant to SPARC
+    query: &organParts |
+      MATCH (n)
+      WHERE n.iri IN [
+          "http://purl.org/sig/ont/fma/fma7195",  // lung
+          "http://purl.org/sig/ont/fma/fma7088",  // heart
+          "http://purl.org/sig/ont/fma/fma7197",  // liver
+          "http://purl.org/sig/ont/fma/fma7198",  // pancreas
+          "http://purl.org/sig/ont/fma/fma7203",  // kidney
+          "http://purl.org/sig/ont/fma/fma7148",  // stomach
+          "http://purl.org/sig/ont/fma/fma7196",  // spleen
+          "http://purl.org/sig/ont/fma/fma14543", // colon
+          "http://purl.org/sig/ont/fma/fma7201",  // large intestine
+          "http://purl.org/sig/ont/fma/fma7200",  // small intestine
+          "http://purl.org/sig/ont/fma/fma7199",  // intestine
+          "http://purl.org/sig/ont/fma/fma15900", // urinary bladder
+          "http://purl.org/sig/ont/fma/fma45659", // lower urinary tract
+          "http://purl.org/sig/ont/fma/fma7157",  // nervous system
+          "http://purl.org/sig/ont/fma/fma9903",  // peripheral nervous system
+          "http://purl.org/sig/ont/fma/fma9906",  // sympathetic nervous system
+          "http://purl.org/sig/ont/fma/fma7647",  // spinal cord
+          "http://purl.org/sig/ont/fma/fma50801"  // brain
+        ]
+      WITH n
+      MATCH path = (start:Class{iri: n.iri})
+      <-[:subClassOf|ilxtr:includedForSPARCUnder|fma:regional_part_of|fma:constitutional_part_of|fma:related_part_of*0..6]-(part)
+      <-[:fma:arterial_supply_of|fma:nerve_supply_of|fma:venous_drainage_of|fma:continuous_with*0..1]-(sup)
+      <-[:subClassOf|fma:constitutional_part_of|fma:branch|fma:tributary|fma:branch_of*0..1]-(a_bit_more)
+      RETURN path
+      UNION  // for this query UNION seems to be MUCH faster than using WITH
```

Comment:
Can you test the `WITH` version of this? The cost of running the `WHERE n.iri IN` query twice probably balances out the cost of using with on the other two. Also it would allow us to avoid the duplication of the list which is already a pain to maintain. I am considering adding an edge to the ontology specifically to generate that list so we don't have to fight with it any more.